modules/test-jpa/src/main/java/com/devonfw/module/test/common/base/ComponentDbTest.java

```diff
@@ -4,21 +4,21 @@
 import javax.persistence.EntityManager;
 import javax.persistence.PersistenceContext;
 
-import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.springframework.test.context.TestExecutionListeners;
 import org.springframework.test.context.junit.jupiter.SpringExtension;
 import org.springframework.test.context.support.DependencyInjectionTestExecutionListener;
 import org.springframework.test.context.transaction.TransactionalTestExecutionListener;
 
+import com.devonfw.module.test.common.api.category.TagComponentTest;
 import com.devonfw.module.test.common.base.clean.TestCleaner;
 
 /**
  * Combination of {@link DbTest} with {@link ComponentTest}.
  */
 @ExtendWith(SpringExtension.class)
 @TestExecutionListeners({ TransactionalTestExecutionListener.class, DependencyInjectionTestExecutionListener.class })
-@Tag("component")
+@TagComponentTest
```

Comment:
Nice approach, did not think of this. Simple string constants might have been easier but this is more sophisticated. Still people who do not want to couple to much with our dependencies can still add stuff `@Tag("component")` manually to their JUnits what was not possible with JUnit 4.